### DISCUSSION TITLE
Problem with Kontakt 7.7 and Audiogridder plugin

### DISCUSSION THREAD

**Discussion by GrandfatherTrout** [apohl79/audiogridder#1295] [Help]
Clicking on Kontakt 7.7 menu bar within the plugin, Audiogridder shuts down/restarts unexpectedly and does not allow the desired menu to be opened/executed.

This behavior can be reproduced and is predictable.
Tested on Mac OS 13.6.1, with Studio One 6.51.

A fix for this would be super cool to have! Thanks for your good work.

**Comment by pradelo** [comment]
Same behaviour on Win10 with cakewalk 2023.09:

- works fine with older versions of kontakt (e.g. 6);
- works bad with 7 .. as suggested might be the grphics of the new library (big squared box) rather than the old one (plugin listed to the left).

Thank you for your work.

**Comment by emack999** [comment]
Kontakt 7 has been very buggy in many areas, across multiple OS.  I'd wait until they actually get it as stable as Kontakt 6 before requesting AG do anything about the issues.
